src/main/java/uk/gov/pay/api/model/CreatePaymentResult.java

```diff
@@ -72,4 +72,11 @@
     @JsonProperty
     @ApiModelProperty(name = "metadata", dataType = "Map[String,String]")
     private ExternalMetadata metadata;
+
+    @JsonProperty
+    @ApiModelProperty(name = "email", example = "citizen@example.org")
```

Comment:
Could be useful to indicate that the fields are not mandatory
```suggestion
    @ApiModelProperty(name = "email", example = "citizen@example.org", required=false)
```